aim/api/resource.py

```diff
@@ -66,6 +85,8 @@ class BridgeDomain(ResourceBase):
     # Attrbutes completely managed by the DB (eg. timestamps)
     db_attributes = []
 
+    _aci_mo_name = 'fvBD'
```

Comment:
One MO name is not always enough (In fact, it is not even for a simple BD). In our case, the Bridge Domain is composed of fvBD and fvRsCtx on APIC which means that two DNs should be extracted from it (and shown by Neutron).

Should we call the converter in order to retrieve the backend identification? This would also have the extra advantage of removing ACI specific information from the AIM library (the converter can eventually be a pluggable component).